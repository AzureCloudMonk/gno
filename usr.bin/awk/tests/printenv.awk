
# Print environment variables' values; names passed on command line

BEGIN {
   if (ARGC < 2) {
      print "Error: provide environment variable name on command line"
      exit 1;
   }
   i = 1;
   do {
      rawname = ARGV[i];
      upname  = toupper(rawname);
      loname  = tolower(rawname);
      printf "%s = '%s'", rawname,ENVIRON[rawname];
      if (rawname != upname)
         printf "; %s = '%s'", upname,ENVIRON[upname];
      if (rawname != loname)
         printf "; %s = '%s'", loname,ENVIRON[loname];
      printf "\n"
      i++;
   } while (i < ARGC)

}
