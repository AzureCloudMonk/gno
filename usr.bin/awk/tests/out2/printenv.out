TestVal = ''; TESTVAL = ''; testval = 'loweronly'
badname = ''; BADNAME = ''
